Ginger/GingerCoreNET/Run/RunListenerLib/CenteralizedExecutionLogger/AccountReportApiHandler.cs

```diff
@@ -104,7 +104,7 @@ public AccountReportRunSet MapDataToAccountReportObject(LiteDbRunSet runSet)
 
         public bool IsConfigurationChanged()
         {
-            return !EndPointUrl.Equals(WorkSpace.Instance.Solution.LoggerConfigurations.CentralLoggerEndPointUrl);
+            return !string.Equals(EndPointUrl, WorkSpace.Instance.Solution.LoggerConfigurations.CentralLoggerEndPointUrl);
```

Comment:
looks like code is doing the same, please explain why this change was needed

Reply:
Changed this code because when the End Point URL is kept empty on the UI in Execution Configuration Tab, EndPointURL becomes null and therefore "EndPointURL.Equals()" code fails and throws null point exception which causes ginger to crash